tests/baselines/reference/user/soap.log

```diff
@@ -0,0 +1,12 @@
+Exit Code: 1
+Standard output:
+node_modules/soap/lib/client.d.ts(4,26): error TS7016: Could not find a declaration file for module 'request'. '../../../tests/cases/user/soap/node_modules/request/index.js' implicitly has an 'any' type.
```

Comment:
they upgraded to typescript, but not to --strict, so we're seeing an error that they are not.

Reply:
I'm not sure what the correct fix is here &mdash; it looks to me like they need `@types/request` in their dependencies, not just devDependencies. Opinions @RyanCavanaugh or @weswigham ?

Reply:
`dependencies` - it's the only way to get `npm i` to install transitive dependencies for consumers (reasonably).